### PR TITLE
Polish TS war factories.

### PR DESCRIPTION
**Pull request by pchote**
Fixes the produced actors clipping through the factory.

### CODE REVIEW

**Comment by abcdefg30** [comment]
Could we also get (something like) `ExitDelay: 3` for the [TD weap](https://github.com/OpenRA/OpenRA/blob/bleed/mods/cnc/rules/structures.yaml#L372), please?

File: mods/ts/rules/nod-structures.yaml [hunk 1]
```diff
@@ -133,8 +133,9 @@ NAWEAP:
 	RallyPoint:
 		RallyPoint: 6,1
 	Exit@1:
-		SpawnOffset: 0,0,0
+		SpawnOffset: -384,-384,0
 		ExitCell: 4,1
+		ExitDelay: 10
```
Comment (abcdefg30) [inline mods/ts/rules/nod-structures.yaml]: I'm not sure if we actually need this for the Nod weap.

File: mods/ts/rules/gdi-structures.yaml [hunk 2]
```diff
@@ -126,8 +126,9 @@ GAWEAP:
 	RallyPoint:
 		RallyPoint: 6,1
 	Exit@1:
-		SpawnOffset: 0,0,0
+		SpawnOffset: -384,-384,0
 		ExitCell: 4,1
+		ExitDelay: 10
```
Comment (abcdefg30) [inline mods/ts/rules/gdi-structures.yaml]: What about something around `3`?

**Comment by pchote** [comment]
3 felt too fast, so I compromised by setting it to 5.  Also fixed the TD delay and overlap.

**Comment by abcdefg30** [comment]
:+1: / :white_check_mark:

**Comment by Mailaender** [comment]
That adds a nice touch-up. :+1:

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/67809429a7adb81b402d6d45cc9e7a591ea3bb31...d1ba377adc50978512044c1c9b3fba6efae039bb)

**Pull request merged**